local/db/activity_program.sql

```diff
@@ -0,0 +1,33 @@
+drop table IF EXISTS activity_program;
+
+create table activity_program (
+  id                 serial primary key,
+  datetime           timestamp primary key,
```

Comment:
idがインクリメントされるので、datetimeのPKは不要かと

```suggestion
  datetime,
```